ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -128,8 +134,10 @@ Verify Displayed GPU Count
     IF    ${expanded}==False
         Click Element    xpath://button[@aria-expanded="false"]/span[.="${server_name}"]
     END
-    Page Should Contain Element    xpath://span[.="${server_name}"]/../../../..//span[.="Number of GPUs"]
-    Page Should Contain Element    xpath://span[.="${server_name}"]/../../../..//span[.="Number of GPUs"]/../../dd/div[.="${no_gpus}"]
+    Click Element    ${SERVING_MODEL_SERVERS_SIDE_MENU}
+    Sleep  5s      reason=wait for ten second until operator goes into init state
```

Comment:
Comment doesn't correspond with the number of seconds in sleep itself, also - is it possible to check other than with sleep?

Reply:
>wait for ten second until operator goes into init state


what operator?


> possible to check other than with sleep?


if you're waiting for a pod, we have different kw to check pods/deployments. Example `Wait For Pods To Be Ready`. Or if you need a different status, you could create another keyword to wait for a specific status

Reply:
My fault. I picked that keyword from another place and forgot to update the reason.
Fixed in [58f7086](https://github.com/red-hat-data-services/ods-ci/pull/971/commits/58f70868872c278508328aff89deadb11a0cc419)